Todo App/app.js

```diff
@@ -34,59 +34,59 @@ function setUser() {
     } else {
         activeUser = userName.toLowerCase(); 
         loadSavedTodos();
-        todoListHeader.innerHTML = "Todo list for " + userName;
+        listHeader.innerHTML = "Todo list for " + userName;
     }
 }
 
 function addTodo(event) {
 
     event.preventDefault();
-    buildTodoList(todoInput.value);
-    saveLocalTodo(todoInput.value);
+    buildTodoList(textInput.value);
+    saveLocalTodo(textInput.value);
     //clear todo input value;
-    todoInput.value = "";
+    textInput.value = "";
 }
 
 // TODO: everything is called "todo" :) this makes reading code quite hard
 // let's use more specific variable names!
 
-// TODO: revise HTML, make it nice and schemantic then adopt JavaScript to the changes
+// TODO: revise HTML, make it nice and semantic then adopt JavaScript to the changes
 function buildTodoList(todo){
-    //prepare the structure by preparing the div and li
-    const todoDiv = document.createElement('div');
-    todoDiv.classList.add("todo");
-
+    //prepare the structure
     const newTodo = document.createElement('li');
     newTodo.innerText = todo;
```

Comment:
the `li` element should contain all necessary elements which belong to a single todo item e.g.:

- text
- complete button
- edit button
- delete button

notice, that `text` would be better off having its own element, rather than setting the `innerHTML` of the `li` itself